lib/helpers/normalizeArgs.js

```diff
@@ -49,4 +47,17 @@ function similarTasks(registry, queryTask) {
   return similarTasks;
 }
 
+function flatten(arr, ret) {
```

Comment:
Should this be `Array.prototype.slice.apply` instead?

Reply:
@phated 
By modifying that way, some test cases failed. As [this case](https://github.com/sttk/undertaker/blob/chore-normalize-repository/test/parallel.js#L83) which is one of such failure test cases, `undertaker` can take arguments like `parallel(['task1', 'task2', 'task3'])`.

Reply:
Sorry, try using `Array.prototype.concat.apply([], args)` to flatten one level deep.